src/cfnlint/rules/resources/CreationPolicy.py

```diff
@@ -32,8 +32,7 @@ def _get_schema(self, resource_type: str) -> dict[str, Any]:
                 "properties": {
                     "StartFleet": {
                         "additionalProperties": False,
```

Comment:
We can get rid of `additionalProperties` too since it only applies to `objects`. This will drop the first error in the 2nd set of tests you changed.  `"{'Type': {}} is not of type 'boolean'",` this would be the important test based on the test case.